addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -25,6 +25,31 @@ def setUpClass(cls):
             signature='--\nMarie'
         )
 
+        records = cls.records
+        partners = cls.partners
+        country_be_id = cls.env.ref('base.be').id
+        _country_us_id = cls.env.ref('base.us').id
+
+        for x in range(10):
```

Comment:
Looks globally like a duplicate of TestMassSMSCommon init, are you adding new records for test purpose ?

Reply:
@tde-banana-odoo yes, is that wrong? i needed more examples to create failed/canceled/clicked and  other traces